src/Analysim.Core/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
@kiyros not sure what this file is. Can you explain?

Reply:
my linter does that automatically when i save it

Reply:
can you change settings so that it's identical to existing code? Otherwise every file you touch will be completely changed.